### ISSUE TITLE
Silktide April 2024 enhancements

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#462]
The Silktide April 2024 report has produced some enhancements to help meet AAA rating. The following should be made a minimum of 44px high:

- Cookie banner 'Hide' button
- Pages in this section navigation links and the accordion heading in mobile
- Breadcrumbs on mobile
- Footer links on mobile (due to smaller font size)
- Call to action button
- Accordion headings
- Contents links in service details pages
